client/fresheyes-web/src/app/components/form/bot.ts

```diff
@@ -15,6 +15,58 @@ const app = new App({
   privateKey: process.env.GITHUB_PRIVATE_KEY ?? "",
 });
 
+export async function getInstalledRepositories(): Promise<{ error: string | null; success: boolean; list: Array<any> | null }> {
+  const session = await auth();
+  if (!session || !session?.user?.login || !session?.accessToken) {
+    return {
+      error: "You must be logged in perform this action",
+      success: false,
+      list: null,
+    };
+  }
+
+  try {
+    const installations = await app.octokit.request("GET /app/installations");
+
+    for (const installation of installations.data) {
+      if (installation.account && installation.app_id === Number(process.env.GITHUB_APP_ID)) {
```

Comment:
`process.env.GITHUB_APP_ID` maybe null. We need to handle the error case of its null or undefined

Reply:
I'll add a fall back in the config files